### PR TITLE
Switch fetch polyfill to official version

### PR DESCRIPTION
**Pull request by nlfurniss**
Now that the github fetch polyfill has pulled down the necessary PRs, we should move away from the forked version

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -5,6 +5,7 @@
 "@ember/test-helpers@^0.7.18":
   version "0.7.25"
   resolved "https://registry.yarnpkg.com/@ember/test-helpers/-/test-helpers-0.7.25.tgz#b4014c108b40ffaf74f3c4d5918800917541541d"
+  integrity sha1-tAFMEItA/69088TVkYgAkXVBVB0=
```
Comment (rwjblue) [inline yarn.lock]: Can you bump your yarn version so we don’t loose all these integrity hashes?

**Comment by nlfurniss** [comment]
Fixed!

**Comment by nlfurniss** [comment]
teeny tiny bump

**Comment by rwjblue** [comment]
ember-cli-pretender 3.1.0 published 🎉

**Pull request merged**